doc/using_tf.rst

```diff
@@ -435,67 +365,617 @@ You can disable MKL-DNN optimization for TensorFlow ``1.8.0`` and above by setti
+
+.. code:: python
+
+  input = {
+    'instances': [1.0, 2.0, 5.0]
+  }
+  result = predictor.predict(input)
+
+The result object will contain a Python dict like this:
+
+.. code:: python
+
+  {
+    'predictions': [3.5, 4.0, 5.5]
+  }
+
+The formats of the input and the output data correspond directly to the request and response formats
+of the ``Predict`` method in the `TensorFlow Serving REST API <https://www.tensorflow.org/serving/api_rest>`_.
+
+If your SavedModel includes the right ``signature_def``, you can also make Classify or Regress requests:
+
+.. code:: python
+
+  # input matches the Classify and Regress API
+  input = {
+    'signature_name': 'tensorflow/serving/regress',
+    'examples': [{'x': 1.0}, {'x': 2.0}]
+  }
+
+  result = predictor.regress(input)  # or predictor.classify(...)
+
+  # result contains:
+  {
+    'results': [3.5, 4.0]
+  }
+
+You can include multiple ``instances`` in your predict request (or multiple ``examples`` in
+classify/regress requests) to get multiple prediction results in one request to your Endpoint:
+
+.. code:: python
+
+  input = {
+    'instances': [
+      [1.0, 2.0, 5.0],
+      [1.0, 2.0, 5.0],
+      [1.0, 2.0, 5.0]
+    ]
+  }
+  result = predictor.predict(input)
+
+  # result contains:
+  {
+    'predictions': [
+      [3.5, 4.0, 5.5],
+      [3.5, 4.0, 5.5],
+      [3.5, 4.0, 5.5]
+    ]
+  }
+
+If your application allows request grouping like this, it is **much** more efficient than making separate requests.
+
+See `Deploying to TensorFlow Serving Endpoints <https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/deploying_tensorflow_serving.rst>` to learn how to deploy your model and make inference requests.
+
+Run a Batch Transform Job
+=========================
+
+Batch transform allows you to get inferences for an entire dataset that is stored in an S3 bucket.
+
+For general information about using batch transform with the SageMaker Python SDK, see :ref:`overview:SageMaker Batch Transform`.
+For information about SageMaker batch transform, see `Get Inferences for an Entire Dataset with Batch Transform <https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html>` in the AWS documentation.
+
+To run a batch transform job, you first create a ``Transformer`` object, and then call that object's ``transform`` method.
+
+Create a Transformer Object
+---------------------------
+
+If you used an estimator to train your model, you can call the ``transformer`` method of the estimator to create a ``Transformer`` object.
+
+For example:
+
+.. code:: python
+
+  bucket = myBucket # The name of the S3 bucket where the results are stored
+  prefix = 'batch-results' # The folder in the S3 bucket where the results are stored
+
+  batch_output = 's3://{}/{}/results'.format(bucket, prefix) # The location to store the results
+
+  tf_transformer = tf_estimator.transformer(instance_count=1, instance_type='ml.m4.xlarge, output_path=batch_output)

```

Comment:
the `^M` that's showing in this diff makes me think something weird is going on with line endings. are you using LF line endings? (CRLF is default on Windows; LF on Mac/Linux)

Reply:
I've set all line endings to LF.